src/Storages/ObjectStorageQueue/ObjectStorageQueueTableMetadata.cpp

```diff
@@ -90,6 +97,7 @@ ObjectStorageQueueTableMetadata::ObjectStorageQueueTableMetadata(const Poco::JSO
     , buckets(getOrDefault(json, "buckets", "", 0))
     , processing_threads_num(getOrDefault(json, "processing_threads_num", "s3queue_", 1))
     , last_processed_path(getOrDefault<String>(json, "last_processed_file", "s3queue_", ""))
+    , loading_retries(getOrDefault(json, "loading_retries", "", 10))
 {
```

Comment:
Can you add modeFromString(mode) here to verify that mode is a correct string?

Reply:
Added